session-replacement/src/main/java/com/amadeus/session/ErrorTracker.java

```diff
@@ -0,0 +1,58 @@
+package com.amadeus.session;
+
+import java.util.concurrent.ConcurrentLinkedQueue;
+
+/**
+ * The class is used to track the number of item during a time.
+ * input parameter
+ *  delay , time in milliseconds where items are keep;
+ *  max: if the number if items is more than the max value the ErrorTracker reachLimits   
+ *
+ */
+
+public class ErrorTracker {
+  public ErrorTracker(int delay , int max) {
+    super();
+    this.delay = delay;
+    this.max = max;
+  }
+
+  private ConcurrentLinkedQueue<Long> list = new ConcurrentLinkedQueue<Long>();
+
+  /**
+   * The class calculate the items during a  time
+   */
+  final int delay ;
+  /**
+   * After the limits the Tracker is considering in error
+   */
+  final int max ;
+  
+  public void addError( long now )  {
```

Comment:
Maybe you could add some javadoc explaining the method?

Reply:
fixed